README.md

```diff
@@ -27,15 +27,56 @@ export COMMAND_STRING_INPUT="convert hdf2splunk -H 127.0.0.1 -u admin -p Valid_p
   - NOTE: This action does not support `view heimdall`.
 
 7. Ensure that the environment variables are set properly: `env`
-8. Deploy the service: `sls deploy --verbose`. This may take several minutes and should look like the example below.
+8. Modify any config values you may want to change. These are found in `config.json` and have the following default values:
+```
+{
+    "service-name": "saf-lambda-function",
+    "bucket-input-folder": "unprocessed/",
+    "bucket-output-folder": "processed/",
+    "output-enabled": true,
+    "output-file-ext": ".json",
+    "output-clarifier": "_output" 
```

Comment:
I would suggest to remove the default values from the Readme.md. Reason being having to remember to update the Readme.md in case the default values are modified in the config.json. Competent people should be able to open config.json and view the default values.

Reply:
`config.json` was deleted because now everything will be configured via environment variables. So there is a `.env.example` that people can reference, but otherwise they will look to the readme for the functionality of how to configure this.